### ISSUE TITLE
[5.x]: Commerce products do not support the `render` function

### ISSUE THREAD

**Issue by white-lukas** [craftcms/commerce#3742]
### Description
When trying to render a commerce product with the `render()` function, the following exception is thrown.
`Element doesn’t have a field layout provider that defines a handle, so it can’t be rendered with a partial template. `

### Steps to reproduce

1. try to render products with the `.render()` function

### Expected behavior
It should render the template inside the `_partials/product/{productType}.twig` file

### Actual behavior
exception thrown

### Craft CMS version

5.4.9

### Craft Commerce version

5.2.1

### PHP version

8.3.12

**Comment by lukeholder** [comment]
Thanks for reporting this. It is fixed for the next release.

To get the fix early, change your `craftcms/commerce` requirement in composer.json to:

```json
"require": {
  "craftcms/commerce": "5.x-dev#84f49807d2bc16d5de69f3ca923d67d7f1fbb60f as 5.2.1",
  "...": "..."
}
```

Then run `composer update`.

We will update this ticket once the release is out.

**Comment by lukeholder** [comment]
This was fixed in 5.2.2 thanks! https://github.com/craftcms/commerce/blob/5.x/CHANGELOG.md#522---2024-11-06
